install.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
@loadams, if I understand correctly, the downsides could break backwards-compatibility since the behavior is different depending on the user environment.

Reply:
@jeffra and @mrwyattii, thoughts on this?

Reply:
@digger-yu and @loadams, thank you both for your clear explanation of this change. It is a good education for me. I am persuaded that this is good change since this script mostly used by power users.